biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1483,12 +1545,26 @@ subroutine UpdateH2OVeg(nsites,sites,bc_out)
           do while(associated(currentCohort))
              ccohort_hydr => currentCohort%co_hydr
              !only account for the water for not newly recruit for mass balance
-             if(.not.ccohort_hydr%is_newly_recruited) then 
+             if(.not.ccohort_hydr%is_newly_recruited) then
+               ! check for nan value , Junyan
+               do ily = 1,csite_hydr%nlevrhiz
```

Comment:
There should be an 'isnan' function somewhere that makes this more obvious. Is it likely to generate nan's still?

Reply:
Working on resolving conflicts with master.  I'm going to remove this in my pass through @JunyanDing .  Let me know if you want to re-address this.